packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -75,8 +77,6 @@ export default class AnalysisWebWorker {
 		this._seoAssessor = null;
 		this._relatedKeywordAssessor = null;
 
-		this.additionalAssessors = {};
```

Comment:
Do we perhaps want to keep this code to add additional assessors, even though we're not using it currently? 🤔